src/js/chapter-factory.js

```diff
@@ -0,0 +1,56 @@
+import { makeDraggable } from "./drag-drop";
+
+const CreateChapter = (text, img) => {
+  const createButtons = () => {
+    const homeButton = document.createElement("button");
+    const nextChapterButton = document.createElement("button");
+
+    homeButton.textContent = "home";
+    nextChapterButton.textContent = "-->";
+
+    return { homeButton, nextChapterButton };
+  };
+
+  const setupImages = (images) => {
+    let gallery = [];
+
+    images.forEach((image) => {
```

Comment:
Maybe would be good to also add a class or id to the images in this image creating loop? And in css there could be defined different classes like .drag-image-1, drag-image-2,...

At the moment you generate the draggable images - dropContainer image is added later, right?

Reply:
Yeah, i had in mind to add classes/id's to images but of course i forgot to do that :D. Later i will add dropContainer.